src/components/AccountSettingsV2.tsx

```diff
@@ -0,0 +1,279 @@
+  //   targetRef.current.scrollIntoView({
+  //     behaviour: 'smooth',
+  //     block: 'start'
+  //   })
+  // }
+  const handleChangesFn = (val: string) => {
+    history.push(`/authenticated/${val}`)
+  }
+  const markFormDirty = () => setChangeInForm(true)
+
+  const unpackBundle = (bundle: UserBundle) => {
+    setUserProfile(bundle.userProfile)
+    setUsername(bundle.userProfile?.userName)
+    setFirstName(bundle.userProfile?.firstName)
+    setLastName(bundle.userProfile?.lastName)
+    setPosition(bundle.userProfile?.position)
+    setCompany(bundle.userProfile?.company)
+    setLocation(bundle.userProfile?.location)
+    setIndustry(bundle.userProfile?.industry)
+    setUrl(bundle.userProfile?.url)
+    setVerified(bundle.isVerified)
+    setOrcid(bundle.ORCID)
+    setIsCertified(bundle.isCertified)
+}
+
+const getUserData = async() => {
+  try{
+    const mask =
+      SynapseConstants.USER_BUNDLE_MASK_ORCID |
+      SynapseConstants.USER_BUNDLE_MASK_USER_PROFILE |
+      SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED |
+      SynapseConstants.USER_BUNDLE_MASK_IS_CERTIFIED |
+      SynapseConstants.USER_BUNDLE_MASK_VERIFICATION_SUBMISSION  
+    const getPrimaryEmail = await SynapseClient.getNotificationEmail(accessToken)
+    const bundle: UserBundle = await SynapseClient.getMyUserBundle(mask, accessToken)
+    setPrimaryEmail(getPrimaryEmail.email)
+    unpackBundle(bundle)
+  } catch (err: any) {
+    displayToast(err.reason as string, 'danger')
+  }
+}
+
+const updateUserProfile = async () => {
+  try{
+    if(userProfile){
+      userProfile.userName = username as string
+      userProfile.firstName = firstName as string
+      userProfile.lastName = lastName as string
+      userProfile.position = position as string
+      userProfile.company = company as string
+      userProfile.location = location as string
+      userProfile.industry = industry as string
+      userProfile.url = url as string
+
+      await SynapseClient.updateMyUserProfile(userProfile, accessToken)
+      getUserData()
+      setChangeInForm(false)
+    }
+  } catch (err: any) {
+    displayToast(err.reason as string, 'danger')
+  }
+}
+
+  useEffect(() => {
+    getUserData()
+  }, []
+  )
+
+  
+  return (
+    <div className='panel-wrapper-bg with-account-setting'>
+      <nav className='account-setting-panel nav-panel'>
+        <Button onClick={() => profileInformationRef.current?.scrollIntoView({behavior:'smooth'})}variant='outlined'>Profile Information</Button> 
+        <Button onClick={() => changePasswordRef.current?.scrollIntoView({behavior:'smooth'})}variant='outlined'>Change Password</Button>
+        <Button onClick={() => trustCredentialRef.current?.scrollIntoView({behavior:'smooth'})}variant='outlined'>Trust & Credentials</Button>
+        <Button onClick={() => personalAccessTokenRef.current?.scrollIntoView({behavior:'smooth'})}variant='outlined'>Personal Access Tokens</Button>
+      </nav>
```

Comment:
Functionality works for scrolling to specific parts of the page. However, it would be ideal if the nav bar would stick to its position.